tests/queries/0_stateless/02361_fsync_profile_events.sql

```diff
@@ -0,0 +1,8 @@
+-- Tags: no-s3-storage
+-- Tag no-s3-storage: s3 does not have fsync
+
+create table data_fsync_pe (key Int) engine=MergeTree() order by key settings min_rows_for_wide_part=2, fsync_after_insert=1, fsync_part_directory=1;
+insert into data_fsync_pe values (1);
+system flush logs;
+select ProfileEvents['FileSync'], ProfileEvents['DirectorySync'], ProfileEvents['FileSyncElapsedMicroseconds']>0, ProfileEvents['DirectorySyncElapsedMicroseconds']>0 from system.query_log where event_date >= yesterday() and current_database = currentDatabase() and query_kind = 'Insert' and type = 'QueryFinish';
```

Comment:
Can it take less than one microsecond?

Reply:
Then it will fail, but seems that it is enough for test right now. Since fsync will do more job usually, than committing single transaction

Reply:
Rewritten to .sh test with retries just in case